docs/_docs/includes/cpp-linux-build-prerequisites.adoc

```diff
@@ -0,0 +1,31 @@
+The following packages should be installed:
```

Comment:
"Should" or "need to be" (aka. must) installed?

Should imply that something is optional.

Reply:
Fixed.

Reply:
Fixed.